rudder-web/src/main/scala/bootstrap/liftweb/Boot.scala

```diff
@@ -148,6 +148,22 @@ class Boot extends Loggable {
     })
 
     ////////// CACHE INVALIDATION FOR RESOURCES //////////
+    // fails on invalid JSON body because it's unsufferable
+    LiftRules.statelessDispatch.append {
+      case req: Req if(   req.json_?
+                       && req.requestType != GetRequest
+                       && req.requestType != HeadRequest
+                       && req.requestType != OptionsRequest
+                       && req.json != null
+                       && req.json.isEmpty
+        ) =>
+        () => Full(
+          JsonResponse(net.liftweb.json.parse(
+            """{"result": "error"
+              |, "errorDetails": "The request has a JSON content type but the body is not valid JSON"}""".stripMargin
```

Comment:
Should we try to display the content ??

Reply:
Let's merge it, better having a v1 :)

Reply:
I don't think so, the user just sent it and we can't even try to tell him where it's not ok